assets/js/googlesitekit/datastore/user/key-metrics.js

```diff
@@ -398,9 +398,9 @@ const baseSelectors = {
 			return widget.modules.every( ( slug ) => {
 				const module = getModule( slug );
 
-				if ( ! module || ! module.connected ) {
+				/*if ( ! module || ! module.connected ) {
 					return false;
-				}
+				}*/
```

Comment:
Since this should be removed let's remove the code instead of commenting it out 🙂

Reply:
This was an oversight, thanks for catching this @tofumatt